packages/playground/src/ui-playground/new-functions.tsx

```diff
@@ -0,0 +1,35 @@
+// For any functions involving shared libraries form controls
+// used throughout the playground
+
+import { RadioButtonOption } from "@batch/ui-react/lib/components/form/radio-button";
+
+/*
+ * Syncs the updated value of a shared libraries text field with a playground component
+ */
+export function _newTextfieldOnChange(
```

Comment:
Is there a reason why these functions are prefixed with an underscore? That's usually the convention for private methods and variables.